app/mailers/organisers_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class OrganisersMailer < ApplicationMailer
+   
+    def organisers_add_email(user)
+      @user = user
+      @url = edit_organiser_url(SecureRandom.hex(8))
```

Comment:
you should create this token while saving organizer and assign it to this user (token - new field in table user)